x-pack/plugins/observability_solution/inventory/public/components/alerts_badge/alerts_badge.tsx

```diff
@@ -15,21 +15,16 @@ export function AlertsBadge({ entity }: { entity: InventoryEntity }) {
   const {
     services: {
       http: { basePath },
-      entityManager,
     },
   } = useKibana();
 
   const activeAlertsHref = basePath.prepend(
     `/app/observability/alerts?_a=${rison.encode({
-      kuery: entityManager.entityClient.asKqlFilter({
-        entity: {
-          identity_fields: entity.entityIdentityFields,
-        },
-        ...entity,
-      }),
+      kuery: `"${entity.entityId}"`,
```

Comment:
Q: So now we just want to have the "id" as filter and not `service.name: "id"` for example?

Reply:
Yes, that's it!